### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

**Comment by gsmet** [comment]
Should we really have `apache` in the name of the repo/artifact? We didn't do it for anything else (Kafka, Tika...). I would avoid it except if there is a very strong reason to do it.

/cc @gastaldi

**Comment by loicmathieu** [comment]
I agree with @gsmet

**Comment by gastaldi** [comment]
@gsmet I followed the `quarkus-apache-httpclient` naming scheme, but we can change that if necessary

File: .github/project.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+release:
+  current-version: 0
```
Comment (gastaldi) [inline .github/project.yml:2]: Merging this PR will trigger the release workflow

**Comment by gastaldi** [comment]
I created https://github.com/quarkiverse/quarkiverse-devops/pull/57 which renames this repo and the team assigned to it

**Comment by gastaldi** [comment]
Voilà! It's https://github.com/quarkiverse/quarkus-pinot now 😉

File: docs/antora.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: quarkus-pinot
+title: Quarkus - Pinot
```
Comment (gastaldi) [inline docs/antora.yml:2]:
```suggestion
title: Quarkus Pinot
```

This will organize it alphabetically

Comment (loicmathieu) [reply docs/antora.yml:2]: This is a generated file, maybe this should be updated on the Quarkiverse template side

Comment (gastaldi) [reply docs/antora.yml:2]: Yeah, I know, it's on my TODO list for a while, it's time to send a PR :)

Comment (gsmet) [reply docs/antora.yml:2]: @gastaldi I wanted to have a look at some of the easy fixes for Quarkiverse generation this afternoon if the release goes well. I will let you know.

File: .github/workflows/quarkus-snapshot.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  #########################
+
+  ECOSYSTEM_CI_REPO_PATH: quarkus-pinot
+
+jobs:
+  build:
+    name: "Build against latest Quarkus snapshot"
+    runs-on: ubuntu-latest
+    # Allow <ADMIN> to manually launch the ecosystem CI in addition to the bots
+    if: github.actor == 'quarkusbot' || github.actor == 'quarkiversebot' || github.actor == '<ADMIN>'
```
Comment (gastaldi) [inline .github/workflows/quarkus-snapshot.yaml:26]:
```suggestion
    if: github.actor == 'quarkusbot' || github.actor == 'quarkiversebot' || github.actor == 'loicmathieu'
```

File: runtime/src/main/java/io/quarkiverse/pinot/runtime/PinotConfig.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class PinotConfig {
+    /**
+     * Zookeeper URL list in the format {host1}:{port1}/{path1},{host2}:{port2}/{path2}.
+     */
+    @ConfigItem
+    public String zookeeperUrls;
```
[the remark this answers is not in the document]
Comment (gastaldi) [reply runtime/src/main/java/io/quarkiverse/pinot/runtime/PinotConfig.java:13]: Meh, the API already expects a single String, maybe that's not necessary for now

File: .github/project.yml (hunk 1 above)
Comment (loicmathieu) [reply .github/project.yml]: I commented the content of the file

**Comment by loicmathieu** [comment]
@gsmet @gastaldi I think it's better now ;)
I'll merge this first impl soon, if you have some feedback don't hesitate !

**Comment by loicmathieu** [comment]
Thanks documentation and integration tests still needs to be done.

**Pull request merged**